UI overhaul.

gdiazc opened the pull request:
bitcoin-sentiments.js:
- Don't use DataTable
- Eliminate scatterplot

index.html:
- Reorder columns,
- Simplify tweet table
- Update jQuery and Bootstrap sources

app.py:
- Remove scatter plot

plotting.py:
- Nicer plot

gdiazc committed:
UI overhaul.

bitcoin-sentiments.js:
- Don't use DataTable
- Eliminate scatterplot

index.html:
- Reorder columns,
- Simplify tweet table
- Update jQuery and Bootstrap sources

app.py:
- Remove scatter plot

plotting.py:
- Nicer plot

gdiazc committed:
UI overhaul continued.

bitcoin-sentiments.js:
- Switch card contents dynamically

index.html:
- Slight UI adjustments

es_client.py:
- Refactor Elasticsearch query to return recent tweets.

gdiazc committed:
Remove .bs_timeline_empty class as soon as request was dispatched to
server, to avoid further requests.

dev1-fund3 commented on static/bitcoin_sentiments.js:
```diff
@@ -18,21 +18,42 @@ angular.module('BitcoinSentimentsApp', [])
+                return response.json()
+            }).then(function(data) {
+                $('#plot_card_spinner').css('display', 'none')
+                Bokeh.embed.embed_item(data['plot2'], 'mainplot_div')
```
Include timezone for the time axis label.

gdiazc replied:
Fixed in [01c0714](https://github.com/fund3/sentiment-dashboard/pull/4/commits/01c0714ebbc5e4495f9523b09bb94b9ea89d1b7a).

dev1-fund3 commented on es_client.py:
```diff
@@ -55,23 +55,23 @@ def get_tweets(client, max_tweets=5000):
-        if len(tweets) <= max_tweets:
-            created_at = _get_with_default(row, 'created_at')
-            full_text = _get_with_default(row, 'full_text')
+    for hit in search:
```
Use list splicing to simplify to `for hit in search[:max_tweets]:` and remove if statement below.

dev1-fund3 reviewed (changes requested):
UI changes look great! Clarification on timezone for axis required and small optimization in for loop.

gdiazc committed:
Add in-house sentiment prediction.

bitcoin_sentiments.js:
- Remove example tweets from UI
- Hide textblob plot for now

app.py:
- Generate in-house sentiment and return plot to client

plotting.py
- polarity_column parameter

requirements.txt
- dill and scikit-learn

sentiments.py (new):
- unpickle in-house trained models and predict

gdiazc committed:
Add notebook

gdiazc committed:
rename vars.

gdiazc committed:
analysis

gdiazc committed:
analysis

gdiazc committed:
Requirements and analysis

gdiazc committed:
Axis formatting

gdiazc committed:
Updated tweet storage.

cron_pull_tweets.py:
- Add ES_TWEET_VERSION to distinguish tweet objects in the ES instance.
- Don't store textblob sentiment in future tweets
- Cleanup

es_client.py:
- Add version to tweet objects
- Add retweet_count to tweet objects

dev1-fund3 reviewed (approved):
Looks good. Let's merge!
